kafka-ui-react-app/src/lib/hooks/useDataSaver.ts

```diff
@@ -1,13 +1,28 @@
 import { isObject } from 'lodash';
+import { alertAdded, alertDissmissed } from 'redux/reducers/alerts/alertsSlice';
+import { useAppDispatch } from 'lib/hooks/redux';
+
+const AUTO_DISMISS_TIME = 2000;
 
 const useDataSaver = (
   subject: string,
   data: Record<string, string> | string
 ) => {
   const copyToClipboard = () => {
```

Comment:
Move dispatch outside copyToClipboard, the hooks should be called only in other hook/component 1st level scope